libs/dev-dock/backend/src/backend.ts

```diff
@@ -0,0 +1,176 @@
+// eslint-disable-next-line vx/gts-no-import-export-type
+import type Express from 'express';
+import * as grout from '@votingworks/grout';
+import * as fs from 'fs';
+import { Optional, assert } from '@votingworks/basics';
+import { safeParseElectionDefinition, UserRole } from '@votingworks/types';
+import { isAbsolute, join } from 'path';
+import {
+  CardStatus,
+  readFromMockFile as readFromCardMockFile,
+} from '@votingworks/auth';
+import {
+  isFeatureFlagEnabled,
+  BooleanEnvironmentVariableName,
+} from '@votingworks/utils';
+import { execFile } from './utils';
+
+export type DevDockUserRole = Exclude<UserRole, 'cardless_voter'>;
+export type DevDockUsbDriveStatus = 'inserted' | 'removed';
+export interface DevDockElectionInfo {
+  title: string;
+  path: string;
+}
+
+function electionPathToAbsolute(path: string) {
+  return isAbsolute(path) ? path : join(__dirname, '../../../..', path);
+}
```

Comment:
It would probably be helpful for future-me to be reminded what relative paths we're expecting here. Relative to the workspace root?

Reply:
Yup, added a comment!